code/modules/projectiles/projectile/bullets/lmg.dm

```diff
@@ -27,22 +27,23 @@
 	name = "7.12x82mm bullet"
 	damage = 40
 	armour_penetration = 5
-	wound_bonus = -50
+	wound_bonus = -40				//hurt a lot but still mostly pointy
 	wound_falloff_tile = 0
 
 /obj/item/projectile/bullet/mm712x82_ap
 	name = "7.12x82mm armor-piercing bullet"
-	damage = 40
-	armour_penetration = 75
+	damage = 35
+	wound_bonus = -45				//they go straight through with little damage to surrounding tissue
+	armour_penetration = 60
+	bare_wound_bonus = -10			//flesh wont stop these very effectively but armor might make it tumble a bit before it enters
 
 /obj/item/projectile/bullet/mm712x82_hp
 	name = "7.12x82mm hollow-point bullet"
 	damage = 50
-	armour_penetration = -60
-	sharpness = SHARP_EDGED
-	wound_bonus = -40
-	bare_wound_bonus = 30
-	wound_falloff_tile = -8
```

Comment:
Undocumented change

Reply:
(Specifically the wound falloff tile)

Reply:
Not in the changelog (which bibby yelled at me for making too long) but it is in the description.